tests/v2/codecoverage/package/Dockerfile

```diff
@@ -165,7 +165,7 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
 
 ENV CATTLE_UI_VERSION 2.7.7-rc1
 ENV CATTLE_DASHBOARD_UI_VERSION v2.7.7-rc1
-ENV CATTLE_CLI_VERSION v2.7.0
+ENV CATTLE_CLI_VERSION v2.7.7-rc1
```

Comment:
We should not change this since it was reverted to be at v2.7.0 earlier due to some provisioning failures found. Lets keep it at v2.7.0